scripts/run_driver_tests.psm1

```diff
@@ -267,4 +267,43 @@ function Invoke-CICDStressTests
     popd
 }
 
+function Invoke-CICDPerformanceTests
+{
+    param(
+        [parameter(Mandatory = $true)][bool] $VerboseLogs,
+        [parameter(Mandatory = $true)][bool] $CaptureProfile)
+    Push-Location $WorkingDirectory
+
+    Write-Log "Executing eBPF kernel mode performance tests."
+
+    $LASTEXITCODE = 0
+
+    # Stop the services, remove the driver from verifier, and restart the services.
+    net.exe stop ebpfsvc
+    net.exe stop ebpfcore
+    # Remove the global verifier settings (this will remove the verifer interceptions that can degrade performance).
+    verifier.exe /volatile 0
+    # Remove the ebpfcore.sys driver from the verifier.
+    verifier.exe /volatile /removedriver ebpfcore.sys
+    net.exe start ebpfcore
+    net.exe start ebpfsvc
+
+    # Extract the performance test zip file.
+    Expand-Archive -Path .\bpf_performance.zip -DestinationPath .\bpf_performance -Force
+    Set-Location bpf_performance
+    # Stop any existing tracing.
+    wpr.exe -cancel
+
+    if ($CaptureProfile) {
+        $pre_command = 'wpr.exe -start CPU'
+        $post_command = 'wpr.exe -stop ""' + $WorkingDirectory + '\bpf_performance_%NAME%.etl""'
+        RelWithDebInfo\bpf_performance_runner.exe -i tests.yml -e .sys -r --pre "$pre_command" --post "$post_command" | Tee-Object -FilePath $WorkingDirectory\bpf_performance_native.csv
+    }
+    else {
+        RelWithDebInfo\bpf_performance_runner.exe -i tests.yml -e .sys -r | Tee-Object -FilePath $WorkingDirectory\bpf_performance_native.csv
```

Comment:
[Nit] As default, this excel .csv format will be nice to have for comparison.

Reply:
What is the question or ask here?

Reply:
Ask: To have .csv as the default behavior.

Reply:
It is the default. Longer term, it will be run through:
https://github.com/Alan-Jowett/bpf_performance/blob/main/scripts/process_results.py 

And then uploaded to a postgres database so it can be analyzed.